doc/mgr/orchestrator_cli.rst

```diff
@@ -6,7 +6,7 @@ Orchestrator CLI
 ================
```

Comment:
could you prefix your commit with `doc/orchestrator` please?

Reply:
Done